types/types.d.ts

```diff
@@ -1995,6 +1995,34 @@ export interface Page {
    */
   exposeFunction(name: string, callback: Function): Promise<void>;
 
+  /**
+   * Sends HTTP(S) request and returns its response. The method will populate request cookies from the context and update
+   * context cookies from the response. The method will automatically follow redirects.
+   * @param urlOrRequest Target URL or Request to get all fetch parameters from.
+   * @param options
+   */
+  fetch(urlOrRequest: string|Request, options?: {
```

Comment:
So, you actually allow passing Request and options together?

Reply:
As discussed off this PR let's use same options for all overloads.